templates/repo/actions/workflow_dispatch.tmpl

```diff
@@ -0,0 +1,72 @@
+<div class="ui ignored info message project-column-header" >
+	<span class="ui left floated text middle">This workflow has a workflow_dispatch event trigger.</span>
+	<button class="ui right floated mini button" onclick="$('#runWorkflowDispatchModal').modal('show');">{{ctx.Locale.Tr "actions.workflow.run"}}{{svg "octicon-triangle-down" 14 "dropdown icon"}}</button>
+</div>
+<div id="runWorkflowDispatchModal" class="ui mini modal">
+	<div class="content">
+		<form id="runWorkflowDispatchForm" class="ui form" action="{{$.Link}}/run?workflow={{$.CurWorkflow}}&actor={{$.CurActor}}&status={{.Status}}" method="post">
+			{{.CsrfTokenHtml}}
+			<div class="ui inline field required">
+				<label>{{ctx.Locale.Tr "actions.workflow.from_ref"}} :</label>
+				<div class="ui floating dropdown icon button">
+					<input type="hidden" name="ref" value="refs/heads/{{index .Branches 0}}">
+					<i>{{svg "octicon-git-branch"}}</i>
+					<span class="text">{{index .Branches 0}}</span>
+					<div class="menu">
+						<div class="ui icon search input">
+							<i class="search icon"></i>
```

Comment:
```suggestion
							{{svg "octicon-search" 16 "search icon"}}
```

Reply:
`<i class="search icon">{{svg "octicon-search" 16 "search icon"}}</i>` This is better